tests/FSharp.Compiler.ComponentTests/CompilerService/FSharpWorkspace.fs

```diff
@@ -49,7 +49,10 @@ type TestingWorkspace(testName) as _this =
 
     interface IDisposable with
         member _.Dispose() =
-            activity.Dispose()
+            if not (isNull activity) then
+                activity.Dispose()
+            else
+                ()
```

Comment:
```suggestion
            use _ = activity
```

Would also work and is null safe just FYI, or you can also call the FSharp.Core primitive for doing disposal.

https://sharplab.io/#v2:DYLgZgzgPg9gDgUwHYAIDKBPCAXBBbAWAChjsNEUANFACgEoUBeYlVlYBbFASySQQBOAEW4QUIFAEkREODAgBDAEYcmKJAFdgwNihZs8+JYJQAPAHQy5EBPSb7drDTZQB9Nb37DRPVPQeOQA